packages/x-tree-view/src/internals/plugins/useTreeViewKeyboardNavigation/useTreeViewKeyboardNavigation.ts

```diff
@@ -182,6 +182,9 @@ export const useTreeViewKeyboardNavigation: TreeViewPlugin<
       // If the focused item is expanded, we move the focus to its first child
       // If the focused item is collapsed and has children, we expand it
       case (key === 'ArrowRight' && !isRtl) || (key === 'ArrowLeft' && isRtl): {
+        if (ctrlPressed) {
```

Comment:
In windows `alt + ArrowLeft or ArrowRight` is used to navigate to prev/ next pages, i've tested in windows, current version looks good in windows hence didn't added any windows specific logic